Cloud Application Manager/Managed Services/getting-started-with-cam-create-managed-application.md

```diff
@@ -0,0 +1,168 @@
+
+![msa-managed-mongodb-scriptbox-vars-defaults.png](../../images/managed-services-anywhere/msa-managed-mongodb-scriptbox-vars-defaults_v1.png)
+
+Here is a figure of the partially expanded events section. Note that **configure** is the only populated event for the parent script box, as it is acting as a kind of mapping between the application script box and the Managed Application script box.
+
+![msa-managed-mongodb-events-expanded.png](../../images/managed-services-anywhere/msa-managed-mongodb-events-expanded_v1.png)
+
+And you can see that I have populated the **configure** event with some code.  Below are the contents of that lifecycle event:
+
+> Bash Example:
+
+```shell
+username="{{ mongo.username }}"
+password="{{ mongo.password }}"
+log_path="{{ mongo.LOG_PATH }}"
+port="{{ mongo.mongodb }}"
+db_path="{{ mongo.DB_PATH }}"
+
+config="{\"username\": \"$username\", \"password\": \"$password\", \"db_path\": \"$db_path\", \"logs\": \"$log_path\", \"db_port\": \"$port\"}"
+
+elasticbox set app.vars.APPLICATION_DESCRIPTION "My managed cross-platform document-oriented database"
+elasticbox set app.vars.APPLICATION_VERSION "{{ mongo.VERSION }}"
+elasticbox set app.vars.APPLICATION_CONFIG "$config"
+```
+
+> Powershell Example (if this were windows):
+
+```powershell
+$username = "{{ mongo.username }}"
+$password = "{{ mongo.password }}"
+$log_path = "{{ mongo.LOG_PATH }}"
+$port = "{{ mongo.mongodb }}"
+$db_path = "{{ mongo.DB_PATH }}"
+
+$config = "{'username': '$username', 'password': '$password', 'db_path': '$db_path', 'logs': '$log_path', 'db_port': '$port'}"
+
+elasticbox set app.vars.APPLICATION_DESCRIPTION "My managed cross-platform document-oriented database"
+elasticbox set app.vars.APPLICATION_VERSION "{{ mongo.VERSION }}"
+elasticbox set app.vars.APPLICATION_CONFIG $config
+```
+
+The above scripts are essentially mapping values from the MongoDB Server script box installation over to the Managed Application script box.  This is also where you might use your custom variables for overriding the default values.
+
+The **APPLICATION_CONFIG** is intended for any additional metadata you wish to provide about the application that would be valuable for its ongoing support.
+
+Also note the **APPLICATION_NAME**; this value is important as it will be used to map this application with its corresponding monitoring policy in the next step.
+
+### Publish your Managed Application
+
+Learn to publish your new Managed Application script box to the public catalog.
+
+This step can be done in parallel to the previous step of creating the monitoring policy.  There is no need to wait for a response from MSA before publishing your script box.
+
+Publishing your script box is explained well at the following location: https://www.ctl.io/knowledge-base/cloud-application-manager/tutorials/publish-script-box/
```

Comment:
Change to:
Publishing your script box is explained well [here](Tutorials/publish-script-box.md).